src/plugins/import-notifications/server/controllers/validateNotifications.ts

```diff
@@ -0,0 +1,17 @@
+import { NotificationRaw } from './parseNotification';
+
+const NOTIFICATIONS_KEYS: (keyof NotificationRaw)[] = ['templateId', 'account', 'data']
```

Comment:
I thought the fields would account as mandatory, and the optional additional columns

The templateId I think we need to show it in the UI in the selector, and when selected would be awesome to show the preview. And after we load the CSV would be good we apply the the substitutions using the first row.

If  everything goes well, the user can do the final upload

I realised this is more work, so no need to this now, but this is how I thought about this feature